### PR TITLE
The Bark Box, new pet toys and squeakie fun.

### PR DESCRIPTION
**Pull request by ItzLollipop**
## About The Pull Request

Adds two different sets of things, frisbees and rubber bones along with  a unique vending machine for pet stuff. 

## Why It's Good For The Game

Adds more fluff items to the game, giving more appeal to certain crowds. 

## Changelog
:cl:
add: Added frisbees, Added rubber bones, and Added a new vendor called bark box
tweak: Tweaked contents between kinkmate and the new bark box
imageadd: Added sprites for both bones and frisbees with alternate and hand held icons.
/:cl:

### CODE REVIEW

**Comment by Tupinambis** [comment]
Just keep it all in one vendor. There is 0 reason for the items to be split between two vendors and it makes mapping harder.

**Review by Ghommie — COMMENTED** [review]
I'd not mind this proposed addition to the game if not for the fact it's aimed to quell certain unorthodox fantasies instead of making pet animals less of a cosmetic feature.
Moreso, this adds new files and lines of code to modular_citadel, when we it should be gradually dismantled instead.
How much these are heavily inspired from tennis balls also disappoint me a little. They could share the same .dmi at least.
Also what Tupinambis said.

**Comment by ItzLollipop** [comment]
Unorthodox fantasies or not, they're fluff items as it is literally aimed for the whole cosmetic part. It's the same kind of concept of using crusader items as a chaplain or even medical doctors having the ability to have a nurses uniform. It adds diversity and in essence allows for people to have more unique characters.
As for the whole modular part, I can switch over and merge it out of the modular folder, and sharing the .dmi files too can be done one hundred percent.
And for the vendor thing, I don't think it would be too realistic to have a kinkmate full of frisbees, balls, or rubber bones. Also, It doesn't necessarily have to have its vendor put into maps as it can be ordered from cargo as any other vendor can be.

**Comment by Owai-Seek** [comment]
I still think it should be added to the kinkmate instead, citadel being a furry server it isnt that far fetched that indulging in pet items would be considered kinky. If in doubt, just make them contraband items so it requires hacking to access them.

**Comment by Ghommie** [comment]
Well, yea, now that I think about it twice, adding frisbees & co to the kinkmate would be far too weird.
Eeeh, go do whatever.

File: code/modules/cargo/packs/costumes_toys.dm [hunk 1]
```diff
@@ -367,3 +367,10 @@
 	contraband = TRUE
 	contains = list(/obj/item/vending_refill/kink, /obj/item/circuitboard/machine/kinkmate)
 	crate_name = "Kinkmate construction kit"
+
+/datum/supply_pack/costumes_toys/barkbox
+	name = "Bark Box construction kit"
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs/costumes_toys.dm:372]: Wrong place, should be placed above the cargo refill crate

File: modular_citadel/code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 2]
```diff
@@ -3,6 +3,11 @@
 	build_path = /obj/machinery/vending/kink
 	req_components = list(/obj/item/vending_refill/kink = 3)
 
+/obj/item/circuitboard/machine/pet
+	name = "BarkBox Vendor (Machine Board)"
+	build_path = /obj/machinery/vending/pet
+	req_components = list(/obj/item/vending_refill/pet = 3)
```
Comment (Trilbyspaceclone) [inline modular_citadel/code/game/objects/items/circuitboards/machine_circuitboards.dm:9]: should be 1 restock

File: modular_citadel/code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 3]
```diff
@@ -3,6 +3,11 @@
 	build_path = /obj/machinery/vending/kink
 	req_components = list(/obj/item/vending_refill/kink = 3)
```
Comment (Trilbyspaceclone) [inline modular_citadel/code/game/objects/items/circuitboards/machine_circuitboards.dm:4]: Can you lower this one to 1 as well?

Comment (ItzLollipop) [reply modular_citadel/code/game/objects/items/circuitboards/machine_circuitboards.dm:4]: Sure thing, I just checked and yeah other vendors do only take one, thanks for pointing that out!

**Pull request closed without merging**